ci-operator/step-registry/redhat-chaos/krkn-hub-console-tests/OWNERS

```diff
@@ -0,0 +1,10 @@
+approvers:
```

Comment:
can we make all these owner files symbolic links to the OWNERs file thats in the redhat-chaos folder so if we have add anyone we dont have to update each file individually

Reply:
example: https://github.com/openshift/release/blob/master/ci-operator/step-registry/openshift-qe/ingress-perf/OWNERS

Reply:
Done, thanks!